x/cronos/keeper/ibc.go

```diff
@@ -52,7 +54,11 @@ func (k Keeper) ConvertVouchersToEvmCoins(ctx sdk.Context, from string, coins sd
 			}
 
 		default:
-			return fmt.Errorf("coin %s is not supported", c.Denom)
+			// TODO use autoDeploy boolean in Params.go
+			err := k.ConvertCoinFromNativeToCRC20(ctx, common.BytesToAddress(acc.Bytes()), c, false)
```

Comment:
Maybe we can enable auto-deploy for now, it would be easier for end-to-end testingthis.
When implementing the gov managed mapping, we can make it controlled by parameter.

Reply:
Good point.
I just notice when writing the test that it is quite hard to test without the auto deploy. I will enable in this PR but we should change it in the TODO so that it can be extracted from the param set